src/main/java/frc/robot/SwerveModule.java

```diff
@@ -86,7 +87,7 @@ public void setDesiredState(SwerveModuleState state) {
     );
 
     // Calculate the turning motor output from the turning PID controller.
-    m_driveMotor.set(driveOutput);
```

Comment:
![Screenshot 2021-11-11 001840](https://user-images.githubusercontent.com/10580424/141264286-c68b5086-fa5e-4cd1-bda8-8e2e1b7b3d82.png)
This was previously expecting values of -1.0 <= x <= 1.0

For TalonFX we need to create the same behavior. We can do a hover over `m_driveMotor.set` or do a control + click to view the class definition they provide us.

![image](https://user-images.githubusercontent.com/10580424/141264709-6d30f417-12ae-4a8d-9f75-bf41795b9d39.png)